mail/gnupg2/Portfile

```diff
@@ -5,6 +5,7 @@ PortSystem          1.0
 set my_name         gnupg
 name                ${my_name}2
 version             2.2.11
+revision            1
```

Comment:
When removing dependencies, increasing the revision is not necessary because the port will still work. No need to have everyone rebuild it.

Reply:
Thanks that's reasonable. Updated.